test/k8s-integration/main.go

```diff
@@ -529,6 +529,7 @@ func runTestsWithConfig(testParams *testParameters, testConfigArg, reportPrefix
 	kubeTest2Args = append(kubeTest2Args, fmt.Sprintf("--skip-regex=%s", skip))
 	kubeTest2Args = append(kubeTest2Args, fmt.Sprintf("--parallel=%d", testParams.parallel))
 	kubeTest2Args = append(kubeTest2Args, fmt.Sprintf("--test-args=%s", testConfigArg))
+	kubeTest2Args = append(kubeTest2Args, "--ginkgo-args=--timeout=24h")
```

Comment:
please put a comment on what timeout this controls

Reply:
Done

Reply:
I don't see any new comments here